### PR TITLE
Improve the support for :lang()

### PR DESCRIPTION
**Pull request by esdoppio**
The scope "meta.language-ranges.css" seems like unnecessary complexity, is it ok to remove it?

### CODE REVIEW

**Comment by esdoppio** [comment]
@50Wliu ready to 🔥

File: grammars/css.cson [hunk 1]
```diff
@@ -561,63 +561,113 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (:)
+          (
+            link|target|visited                                     # location
+          | active|focus|hover                                      # user action
+          | default|disabled|enabled|read-only|read-write           # input control state
+          | checked|indeterminate                                   # input value state
```
Comment (50Wliu) [inline grammars/css.cson]: Looks like we lose the extra `.ui-state` class here.  Do you think that's important?

Comment (esdoppio) [inline grammars/css.cson]: As I said in the comment of ee52174, this name "ui-state" itself doesn't make sense, that's why I want to remove it. And I think it should be safe even for themes that are relying on this class coz the related properties will still get the "pseudo-class" class.

File: grammars/css.cson [hunk 2]
```diff
@@ -561,63 +561,113 @@
...
             'name': 'punctuation.section.function.css'
+        'contentName': 'meta.language-ranges.css'
         'end': '\\)'
         'endCaptures':
           '0':
             'name': 'punctuation.section.function.css'
         'patterns': [
           {
-            'include': '#selector'
+            'match': '(?i)(?<=[(,\\s])[a-z]+(-[a-z0-9]*+)*+(?=[),\\s])'
```
Comment (esdoppio) [inline grammars/css.cson]: Yeah, that would be better 😆

**Comment by esdoppio** [comment]
Do you think "meta.language-ranges.css" is necessary here?

**Comment by 50Wliu** [comment]
Thought I commented about that.  I guess not.

No, I don't think it's necessary.  This isn't really something that can be autocompleted.

**Pull request merged**